src/microbe_stage/editor/PatchMapNode.tscn

```diff
@@ -84,5 +87,18 @@ label_settings = ExtResource("4_tygy2")
 horizontal_alignment = 1
 vertical_alignment = 1
 
+[node name="Margin" type="VBoxContainer" parent="."]
+custom_minimum_size = Vector2(16, 64)
+layout_mode = 2
+size_flags_vertical = 0
+
+[node name="Eruption" type="TextureRect" parent="Margin"]
```

Comment:
Would be pretty nice to have a tooltip explaining on hover what the icon means. Something like `EVENT_TOOLTIP_ERUPTION` with text like 
```
Eruption Patch Event
Creates extra hydrogen sulfide and carbon dioxide
```